src/components/Header/Header.tsx

```diff
@@ -54,25 +55,35 @@ const PrimaryCta = ({ children, className = '' }: { children: any; className?: s
 
 export const Header = ({
     onPostPage,
+    onHomePage = false,
     transparentBackground = false,
+    onBlogPage = false,
+    isBlogArticlePage = false,
 }: {
     onPostPage: boolean
+    onHomePage?: boolean
+    onBlogPage?: boolean
     transparentBackground?: boolean
+    isBlogArticlePage?: boolean
 }) => {
     const [expanded, expandMenu] = useState(false)
     const { websiteTheme } = useValues(layoutLogic)
 
-    const themeSupportedColor = websiteTheme === 'light' ? 'bg-lightmode-gray' : 'bg-darkmode-purple'
-    const backgroundColor = onPostPage ? themeSupportedColor : 'bg-purple-gradient'
-    const logo = (onPostPage || transparentBackground) && websiteTheme === 'light' ? darkLogo : whiteLogo
-    const textLight = (!onPostPage && !transparentBackground) || websiteTheme === 'dark'
+    const logo = onPostPage && websiteTheme === 'light' && !isBlogArticlePage ? darkLogo : whiteLogo
+    const textLight =
```

Comment:
this is a mess I'd like to refactor, but won't do it in this PR. My 🍽️ is already full enough